e2e/__tests__/snapshot.test.js

```diff
@@ -29,7 +29,10 @@ const originalTestPath = path.resolve(
   '../snapshot/__tests__/snapshot.test.js',
 );
 const originalTestContent = fs.readFileSync(originalTestPath, 'utf8');
-const copyOfTestPath = originalTestPath.replace('.js', '_copy.js');
+const copyOfTestPath = path.resolve(
```

Comment:
```suggestion
const copyOfTestPath = originalTestPath.replace(/\.js$/, '_copy.js');
```

I find it's easier to understand the intention then